src/AuditManager.php

```diff
@@ -44,14 +48,19 @@ public function getConfiguration()
         return $this->config;
     }
 
-    public function createAuditReader(EntityManager $em)
+    public function createAuditReader(?EntityManager $em = null)
```

Comment:
This change breaks BC.

Reply:
how?
u can send it or not .. how will it break BC?